src/client/routes/Disclaimer/index.tsx

```diff
@@ -1,16 +1,47 @@
 import styled from 'styled-components';
 
```

Comment:
Much better disclaimer nice, let me know if we need to translate it to other locales or just EN is ok

Reply:
oh yes this will need translating